### PR TITLE
fix: errors in plunkr template

### PR DESCRIPTION
**Pull request by crisbeto**
* Fixes errors in the Plunkr template due to outdated Material version.
* Updates the outdated theme link.
* Disables the opening of the Plunk in a new tab, because it gets blocked by the browser as a popup.

This is based on the Plunkr from our issue template.

### CODE REVIEW

**Comment by crisbeto** [comment]
@jelbourn a couple of notes:

1. This template will load the latest build of Material, however the Angular version are locked into the ones from NPM. Is this what we want? Otherwise I can switch back to loading from NPM.
2. I disabled the opening in a new tab since the browser considers it a popup and blocks it. I have a workaround which is hacky and will likely break. I'm looking for a better workaround.

File: src/assets/plunker/index.html [hunk 1]
```diff
@@ -23,10 +23,9 @@
   </script>
 
   <!-- Load the Angular Material 2 stylesheet -->
-  <link href="https://unpkg.com/@angular/material/core/theming/prebuilt/indigo-pink.css" rel="stylesheet">
-
+  <link href="https://cdn.rawgit.com/angular/material2-builds/master/core/theming/prebuilt/indigo-pink.css" rel="stylesheet">
```
Comment (DevVersion) [inline src/assets/plunker/index.html]: Using the CDN for rawgit when loading the `master` branch may cause issues.

See https://github.com/rgrove/rawgit/issues/51

File: src/assets/plunker/systemjs.config.js [hunk 2]
```diff
@@ -28,24 +16,24 @@ System.config({
...
-    '@angular/platform-browser-dynamic': 'vendor/@angular/platform-browser-dynamic/bundles/platform-browser-dynamic.umd.js',
+    '@angular/core': 'http://unpkg.com/@angular/core/bundles/core.umd.js',
+    '@angular/common': 'http://unpkg.com/@angular/common/bundles/common.umd.js',
+    '@angular/compiler': 'http://unpkg.com/@angular/compiler/bundles/compiler.umd.js',
+    '@angular/http': 'http://unpkg.com/@angular/http/bundles/http.umd.js',
+    '@angular/forms': 'http://unpkg.com/@angular/forms/bundles/forms.umd.js',
+    '@angular/router': 'http://unpkg.com/@angular/router/bundles/router.umd.js',
+    '@angular/platform-browser': 'http://unpkg.com/@angular/platform-browser/bundles/platform-browser.umd.js',
+    '@angular/platform-browser-dynamic': 'http://unpkg.com/@angular/platform-browser-dynamic/bundles/platform-browser-dynamic.umd.js',
+    '@angular/material': 'https://cdn.rawgit.com/angular/material2-builds/master/bundles/material.umd.js',
```
Comment (DevVersion) [inline src/assets/plunker/systemjs.config.js]: Same here with the `cdn`. (https://github.com/rgrove/rawgit/issues/51)

File: src/assets/plunker/systemjs.config.js [hunk 3]
```diff
@@ -27,7 +27,7 @@ System.config({
     '@angular/router': 'http://unpkg.com/@angular/router/bundles/router.umd.js',
     '@angular/platform-browser': 'http://unpkg.com/@angular/platform-browser/bundles/platform-browser.umd.js',
     '@angular/platform-browser-dynamic': 'http://unpkg.com/@angular/platform-browser-dynamic/bundles/platform-browser-dynamic.umd.js',
-    '@angular/material': 'https://cdn.rawgit.com/angular/material2-builds/master/bundles/material.umd.js',
+    '@angular/material': 'https://unpkg.com/@angular/material/core/theming/prebuilt/indigo-pink.css',
```
Comment (DevVersion) [inline src/assets/plunker/systemjs.config.js]: This should be actually the `.umd` bundle 😄

**Comment by jelbourn** [comment]
LGTM

**Pull request merged**